include/albatross/src/indexing/group_by.hpp

```diff
@@ -80,7 +80,7 @@ template <typename KeyType, typename ValueType> class GroupedBase {
 
   std::pair<KeyType, ValueType> first_group() const { return *map_.begin(); }
 
-  ValueType first_value() const { return map_.begin()->first; }
+  ValueType first_value() const { return map_.begin()->second; }
```

Comment:
Is there a test that would've caught this sooner?

Reply:
good call.